engines/wintermute/ad/ad_actor_3dx.cpp

```diff
@@ -489,9 +487,14 @@ bool AdActor3DX::displayShadowVolume() {
 			continue;
 		}
 
-		at->displayShadowVol(*boneMat, lightVector, extrusionDepth, true);
+		Math::Matrix4 transposedWorldMatrix = _worldMatrix;
+		transposedWorldMatrix.transpose();
+
+		at->displayShadowVol(transposedWorldMatrix * *boneMat, lightVector, extrusionDepth, true);
```

Comment:
I would consider putting parentheses around *boneMat here, to make it slightly prettier.

Reply:
()'s added.